src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -115,11 +141,58 @@ public static void deleteTempTargetBQTable() throws IOException, InterruptedExce
     }
   }
 
+  @After(order = 1, value = "@BQ_SINK_TEST_CSV_BQMT")
+  public static void deleteTempTargetBQMTCSVTables() throws IOException, URISyntaxException, InterruptedException {
+    deleteTempTargetBQTables(PluginPropertyUtils.pluginProp("bqmtCsvTargetTables"));
+  }
+
+  public static void deleteTempTargetBQTables(String tableList) throws IOException, InterruptedException {
```

Comment:
Change to `private`.

And move this method.. add after `@After(order = 1, value = "@BQ_SINK_TEST_SPLIT_BQMT")`

Reply:
Change done.